lib/pure/times.nim

```diff
@@ -880,6 +880,11 @@ since((1, 1)):
   export fromUnixFloat
   export toUnixFloat
 
+
+when defined(windows): # TODO put to a separate module
```

Comment:
move to winlean, where getSystemTimeAsFileTime is?

Reply:
make sense